src/components/Quiz.jsx

```diff
@@ -1,48 +1,70 @@
-import { useState } from "react"
-import QUESTIONS from '../question.js'
+import { useState, useCallback } from "react"
+import FIRSTQUIZ from '../firstQuiz.js'
+import SECONDQUIZ from '../secondQuiz.js'
+import THIRDQUIZ from '../thirdQuiz.js'
 import quizComplete from '../assets/quiz-complete.png'
+import QuestionTimer from "./QuestionTimer.jsx";
 
+const quizzes = [FIRSTQUIZ, SECONDQUIZ, THIRDQUIZ]
 
 export default function Quiz() {
 
-    const [userAnswer, setUserAnswer] = useState([]); // kullanıcının cevaplarını da işlememiz lazım vr dom da
-    const activeQuestionIndex = userAnswer.length; // indexe bakarak bir sonraki sorunun sorulacağını anlayabiliriz ve daha az durum yönetimi yapmış oluruz 
+    const [activeQuizIndex, setActiveQuizIndex] = useState(0) // hangi sınacın faal olduğunu index ile takip edeceğiz 
+    const [userAnswer, setUserAnswer] = useState([]);
+    const activeQuiz = quizzes[activeQuizIndex]
```

Comment:
use initial values for fallback https://stackoverflow.com/questions/5409641/set-a-variable-if-undefined-in-javascript

const activeQuiz = quizzes[activeQuizIndex] || [];